src/debian/vmmgrapi/gunicorn.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=gunicorn daemon
+Requires=gunicorn.socket
+After=network.target
+
+[Service]
+Type=notify
+# the specific user that our service will run as
+User=root
```

Comment:
Is it possible to not launch gunicorn with another user than root?

Reply:
for now, vm-mgr requires the root user (needs to run some crm and rbd commands)